Assets/UnityIO.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public class UnityIO
+{
+	public class Response {
+		public string error;
+		public byte[] data;
+
+		public Response(string e, byte[] d) {
+			error = e;
+			data = d;
+		}
+		public Response(string e, string d) {
+			error = e;
+			data = Convert.FromBase64String(d);
```

Comment:
Let me rephrase: Why does this class need to be aware of how the response data is encoded? Decoding the response is the responsibility of the code that made the request, right?